### ISSUE TITLE
Is the repo still maintained?

### ISSUE THREAD

**Issue by rimonhanna** [NotionX/react-notion-x#544]
Is the repo still maintained? Should we start a fork and maintain that instead?

**Comment by Kuriel23** [comment]
It seems deprecated a year ago.

**Comment by transitive-bullshit** [comment]
I no longer have time to properly maintain this project.

Previously, @normdoow was helping to maintain it, but I believe he sold the company he was working on which depended on `react-notion-x` and is no longer available as a maintainer.

I'm very open to requests from people who are serious about maintaining this project. Please respond to this thread and include a list of PRs that you've opened as evidence that you'd make a good maintainer, and I'll add you as collaborator permissions to the repo.

Thanks && sorry, but that's just the way that free OSS tends to go over time.

**Comment by normdoow** [comment]
Yeah, I have moved on from the Notion space. Someone should totally take this on!

Thanks @transitive-bullshit!

**Comment by transitive-bullshit** [comment]
@rimonhanna that'd be awesome.

If you'd like collaborator & NPM access, please take a look through any existing PRs you'd like to prioritize and open a general maintenance PR (mainly updating deps), and tag me explicitly in any of them to make sure they're on my radar. For now, I'll try my best to get back to you asap, and if that goes well, then I'll be happy to add permissions for you.

Thanks!!! 🙏

**Comment by jesusdmedinac** [comment]
I would love to have collaborator & NPM access to support the lib. I'm trying to fix the search tool but I see you already fix it but the 6.16.2 version was not published.

**Comment by onnimonni** [comment]
@transitive-bullshit I would be happy to go through the PRs and issues and help out with the project.

I have mainly been creating PRs for the https://github.com/transitive-bullshit/nextjs-notion-starter-kit but I would be willing to give extra hand here as well.

**Comment by onnimonni** [comment]
## Update about project status
Just to update rest of you here I was able to talk to @transitive-bullshit with email and he's happy to get extra hands maintaining this project.

I documented a working local environment in https://github.com/NotionX/react-notion-x/pull/568.

If you will use the installation steps from there you should be able to develop this project locally.

It's at least working for me now with MacOS 15.1 with Apple silicon.

## Help wanted
If you do want to make this project succeed in the long run I added couple of ideas which would help:

### Updating supported node version and dependencies to newer versions
It would be great to get small pull requests into the project which would help us first to move from Node 16->18 and then from 18->20 and 20->22.

This of course will require updating some dependencies as well.
